apps/services/models.py

```diff
@@ -193,3 +193,11 @@ def __repr__(self):
                                   Column('commandant_full_name', JSON),
                                   Column('telephone_number', VARCHAR(50)),
                                   Column('documents', JSON))
+
+
+return_user_document_view = Table('return_user_document_view', metadata_obj,
```

Comment:
u'll get an error too soon bcs the columns order of your SQL view, sqlalchemy view obj and out schema aren't the same. Please make it all identical